reference-files/token-map.json

```diff
@@ -0,0 +1,5 @@
+{
+	"text.primary": "--primary-700",
+	"text.secondary": "--accent1-500",
```

Comment:
Picking these values based on looking at the Design Tokens defined in [here](https://designsystem.webstandards.ca.gov/style/tokens/#color-tokens). The idea is to take a css variable and map it to its corresponding theme.json token.

Reply:
Should we add some explanatory text on how you got to `text.primary` looks like it is based on the nesting under `custom`.

I like that it makes this object one level deep, but took me a second to figure out what's what.

Reply:
Yip, I was planning to update the README.md with how this was crafted since you can't put comments here.